src/status_im/contexts/keycard/effects.cljs

```diff
@@ -0,0 +1,110 @@
+     (keycard/on-card-disconnected #(rf/dispatch [:keycard.callback/on-card-disconnected]))
+     (when platform/ios?
+       (keycard/on-nfc-user-cancelled #(rf/dispatch [:keycard.ios.callback/on-nfc-user-cancelled])))
+     (when platform/ios?
+       (keycard/on-nfc-timeout #(rf/dispatch [:keycard.ios.callback/on-nfc-timeout])))
+     (keycard/on-nfc-enabled #(rf/dispatch [:keycard.callback/check-nfc-enabled-success true]))
+     (keycard/on-nfc-disabled #(rf/dispatch [:keycard.callback/check-nfc-enabled-success false]))]))
+(rf/reg-fx :effects.keycard/register-card-events register-card-events)
+
+(defn check-nfc-enabled
+  []
+  (log/debug "[keycard] check-nfc-enabled")
+  (keycard/check-nfc-enabled
+   {:on-success
+    (fn [response]
+      (log/debug "[keycard response] check-nfc-enabled")
+      (rf/dispatch [:keycard.callback/check-nfc-enabled-success response]))}))
+(rf/reg-fx :effects.keycard/check-nfc-enabled check-nfc-enabled)
+
+(rf/reg-fx
+ :effects.keycard.ios/start-nfc
+ (fn [{:keys [on-success on-failure]}]
+   (log/debug "fx start-nfc")
+   (keycard/start-nfc {:on-success (or on-success #()) :on-failure (or on-failure #())})))
+
+(rf/reg-fx
+ :effects.keycard.ios/stop-nfc
+ (fn [{:keys [on-success on-failure]}]
+   (log/debug "fx stop-nfc")
+   (keycard/stop-nfc {:on-success (or on-success #()) :on-failure (or on-failure #())})))
+
+(defn- error-object->map
+  [^js object]
+  {:code  (.-code object)
+   :error (.-message object)})
+
+(defn get-application-info
+  [{:keys [on-success on-failure] :as args}]
+  (log/debug "[keycard] get-application-info")
+  (keycard/get-application-info
+   (merge
+    args
+    {:on-success
+     (fn [response]
+       (log/debug "[keycard response succ] get-application-info")
+       (when on-success
+         (on-success response)))
+     :on-failure
+     (fn [response]
+       (log/debug "[keycard response fail] get-application-info")
+       (when on-failure
+         (on-failure (error-object->map response))))})))
+(rf/reg-fx :effects.keycard/get-application-info get-application-info)
+
+(defn get-keys
+  [{:keys [on-success on-failure] :as args}]
+  (log/debug "[keycard] get-keys")
+  (keycard/get-keys
+   (assoc
+    args
+    :on-success
+    (fn [response]
+      (log/debug "[keycard response succ] get-keys")
+      (when on-success
+        (on-success response)))
+    :on-failure
+    (fn [response]
+      (log/warn "[keycard response fail] get-keys"
+                (error-object->map response))
+      (when on-failure
+        (on-failure (error-object->map response)))))))
+(rf/reg-fx :effects.keycard/get-keys get-keys)
+
+(defn login
+  [{:keys [key-uid password whisper-private-key]}]
+  (native-module/login-account
+   (assoc (profile.config/login)
+          :keyUid                   key-uid
+          :password                 password
+          :keycardWhisperPrivateKey whisper-private-key)))
+(rf/reg-fx :effects.keycard/login-with-keycard login)
+
+(defn retrieve-pairings
```

Comment:
idk honestly just saw a few new effects, and decided we have a new guideline :) I believe there should be no big difference

Reply:
No worries. There's no guideline. What you probably saw are events being declared separately as vars because the function needs to be called directly in unit tests. But since a PR I opened not long ago, we are now able to write unit tests for events without having to define function vars.

In the case here of this `effects` namespace, I think it would be a bit better if you defined the functions anonymously, just because it's what we do normally in the rest of the repo.